src/radio/index.js

```diff
@@ -87,6 +89,7 @@ class Radio extends EventEmitter {
     this.current = null;
     this.history = [];
     this.skips = new Set();
+    this.TaskRunner = new TaskRunner();
```

Comment:
One thing we can do is move these up to the type definitions. We're doing that in `TaskRunner` already.